src/pages/HomePage.tsx

```diff
@@ -42,7 +42,7 @@ const HomePage: FC<IProps & RouteComponentProps<IRouteProps>> = ({ match }) => {
       <Box flexGrow={5} className={classes.widgetBox} position="relative">
         <OverlayLoader show={loading} />
         <FilterBar />
-        <WidgetsTemplate id={id} />
+        {id ? <WidgetsTemplate id={id} /> : <Redirect to={`/newsflash/${initialId}`} />}
```

Comment:
when using redirect in a component with view - please separate between rendered return to redirect. like:
```
if(something) {
  return <Redirect to={`/newsflash/${initialId}`} />
}

return (
  <div>Look at this amazing view!</div>
)
```